extensions/amp-date-display/0.2/stories/Basic.stories.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DateDisplay} from '../date-display';
+import {date, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+// TODO(wassgha): Figure out what to do with JSX
+import {h} from 'preact'; // eslint-disable-line
```

Comment:
@dvoytenko / @jridgewell same here

Reply:
No idea. But we can probably live with this for some time?

Reply:
Why are we doing this?

Reply:
Something somewhere is complaining about `h` being undefined because we're using `jsx` in these files. I'm assuming `h` is probably the tag function that processes jsx?